### PR TITLE
Refactor: Nipple suction cups to extended items

### PR DESCRIPTION
**Pull request by SandrineBC**
# Summary
This PR refactors the nipple suction cups to use the extended item script and purges a small layout error in the extended item button placement for four options

# Dependencies
This PR needs #1436 for the `ShowImage` option of the extended item dialogs

### CODE REVIEW

File: BondageClub/Screens/Character/Player/Dialog_Player.csv [hunk 1]
```diff
@@ -136,11 +136,17 @@ InflateLevel1,,,Inflation level: Light,,
 InflateLevel2,,,Inflation level: Inflated,,
 InflateLevel3,,,Inflation level: Bloated,,
 InflateLevel4,,,Inflation level: Max,,
+SelectSuctionLevel,,,Select suction level,,
 SuctionLevel0,,,Suction Cup level: Loose,,
```
[the remark this answers is not in the document]
Comment (SandrineBC) [reply BondageClub/Screens/Character/Player/Dialog_Player.csv:140]: Yes, that’s on my todo list. Same for the inflatable items.

[the remark this answers is not in the document]
Comment (SandrineBC) [reply BondageClub/Screens/Character/Player/Dialog_Player.csv:140]: Yes, that’s on my todo list. Same for the inflatable items.

**Review by Nina-1474 — APPROVED** [review]

File: BondageClub/Screens/Character/Player/Dialog_Player.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (elliesec) [reply BondageClub/Screens/Character/Player/Dialog_Player.csv:140]: I think now would be a good time to remove those `SuctionLevel<Number>` entries, then I think this is good to go in.

**Review by elliesec — CHANGES_REQUESTED** [review]

File: BondageClub/Screens/Character/Player/Dialog_Player.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (SandrineBC) [reply BondageClub/Screens/Character/Player/Dialog_Player.csv:140]: PR #1440 should go in, before. Then this should be the last one...

**Review by elliesec — APPROVED** [review]

**Pull request merged**